addon-test-support/ember-qunit/index.js

```diff
@@ -228,6 +232,28 @@ export function setupEmberOnerrorValidation() {
   });
 }
 
+export function setupAsyncTimerLeakDetection() {
+  QUnit.testDone(({ module, name }) => {
```

Comment:
making these callbacks separate (but private) exported functions will allow you to invoke them manually from within a single test and assert that the expected error is thrown....

Reply:
Sounds good. That will address my nagging guilt at the thought of pushing code with no tests!

Reply:
lol, its the guilt that drives me...

Reply:
Split things up. Added tests.